gini/gini-vision-lib-ios
# Feature flash mode

### Description
Added flash toggle in the camera screen. It'll be positioned differently depending on the device, being like this:

![image](https://user-images.githubusercontent.com/9082270/53013540-52062400-3446-11e9-8fa7-594dd098f86c.png)
 
![image](https://user-images.githubusercontent.com/9082270/53013589-719d4c80-3446-11e9-8e92-fdb5de9dbf4c.png)

### How to test
Run the example app and check that all the camera buttons are shown. Change the `GiniConfiguration` properties to see that the buttons position change.

### Merging
Automatic

### Issues
